### PR TITLE
Improve Cargo Network Trafic

### PR DESCRIPTION
**Pull request by jokoho48**
This PR reduce the Network Trafic because if you attachTo something its get Synced every Frame and setPos is more static and dont create less Trafic

### CODE REVIEW

**Comment by PabstMirror** [comment]
Unloaded object is invulnerable.

**Comment by esteldunedain** [comment]
> if you attachTo something its get Synced every Frame

Was this confirmed? If it's implemented properly in the engine it should not cause extra updates.

File: addons/cargo/functions/fnc_loadItem.sqf [hunk 1]
```diff
@@ -35,7 +35,8 @@ _vehicle setVariable [QGVAR(space), _space - _itemSize, true];
 
 if (_item isEqualType objNull) then {
     detach _item;
-    _item attachTo [_vehicle,[0,0,-100]];
+    _item setPos [-100000,-100000,-100000];
+    _item allowDamage false;
```
Comment (Glowbal) [inline addons/cargo/functions/fnc_loadItem.sqf]: I do not like this. A better approach might be to add a `handleDamage` eventhandler and return current damage instead

**Comment by Glowbal** [comment]
Merge conflicts and using allowDamage. We have no issues reported with the current approach.

**Pull request closed without merging**